### ISSUE TITLE
"Reset Camera for Graphics Window ('r')" does not work in 3D

### ISSUE THREAD

**Issue by jpsluka** [CompuCell3D/cc3d-player5#29]
"Reset Camera for Graphics Window ('r')" works in 2D but not in 3D. It looks like the x,y,z offsets and zoom level get reset, but the rotations do not. The 'r' key does the same thing, resets position and zoom, but not rotations.

**Comment by maciekswat** [comment]
Yes, this is true. We only reset the zoom position, not the rotation. It might be a good idea to add full reset, as you described it
